scripts/convert-platform-test-data.js

```diff
@@ -0,0 +1,82 @@
+"use strict";
+// Converts old urltestdata.txt format to new urltestdata.json format, and also produces some intermediate files for
```

Comment:
I guess we don't. I used this while doing the switch and thought it might be good to keep around, but we can get rid of it.